firmware/Core/Src/Controller.c

```diff
@@ -28,6 +28,7 @@ static ActuatorCommands_t ActuatorCommands;
 
 static Controller_State_t currentState;
 static bool periodHasPassed;
+static temp_probe_readings readings;
```

Comment:
We're cutting it so close now that it doesent matter, but it would be nice to keep up with the same convention as the other data struct. Anyway, no big deal.

Reply:
updated to match naming convention